CONTRIBUTING.md

```diff
@@ -75,7 +76,13 @@ export PATH="<path_to_torchaudio>/third_party/install/bin:${PATH}"
 The following dependencies are also needed for testing:
 
 ```bash
-pip install typing pytest scipy numpy parameterized
+pip install typing pytest scipy numpy parameterized pyyaml
```

Comment:
`pyyaml` is not required. It is required when you use `fairseq`.

Reply:
I've removed pyyaml completely since we are not including fariseq for now.